docs/documentation/telemetry.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: documentation
+title: Telemetry
+doc_order: 100
+description: About the use of Telemetry in mozregression
+---
+
+# Telemetry
+
+Unless explicitly disabled, mozregression will send an anonymized telemetry ping (a small JSON-document) to Mozilla's servers on each use (command-line invocation in the case of console mozregression, each bisection or single run in the case of the GUI).
+
+This data is anonymized and not personally identifiable, it contains only:
+
+- A random client id (e.g. `5a26b9f7-c025-4858-abce-63ca73ceebf0`)
+- Some very basic information on the environment in which mozregression
+  is running (e.g. operating system)
+- Some other metrics designed to measure and understand the nature of
+  usage of mozregression (for example, which applications are being bisected) -- for an up-to-date list, please see this [automatically
+  generated report](https://github.com/mozilla/mozregression/blob/master/docs/glean/metrics.md).
+
+The purpose of this data gathering is only to improve mozregression itself by understanding the scope of its environment and usage, and
```

Comment:
This sould really go into the `pings.yaml` ping description, so that it shows up in the autogenerated docs.